net-mail/notmuch/notmuch-0.36.ebuild

```diff
@@ -0,0 +1,302 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_OPTIONAL=1
+NEED_EMACS="24.1"
+PYTHON_COMPAT=( python3_{7..10} pypy3 )
```

Comment:
remove 3.7, add 3.11 (if tests pass ofc)

Reply:
Removed 3.7. But I'm not sure how to emerge 3.11 on my system. I'll try again tomorrow, but if you have a link for install docs, I'd appreciate it.